docs/source/whatsnew/version8.rst

```diff
@@ -392,16 +392,29 @@ Reverted in 8.17.1:
 
   - :ghpull:`14190` remove support for python 2 in lexers (reverted in 8.17.1 as it is imported by qtconsole/spyder)
 
-Mamba and Micromamba magic
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. _ref: mamba-and-micromamba-magic
+.. _ref: mamba-and-micromamba-magic-commands
```

Comment:
I'm going to remove this as this is not the syntax to add references to header and it make the doc build fail.

```suggestion
```